### PR TITLE
Feature/dancing goat v28

### PR DESCRIPTION
**Pull request by bkapustik**

### CODE REVIEW

File: tests/Kentico.Xperience.Lucene.Tests/Usings.cs
Comment (seangwright) [inline tests/Kentico.Xperience.Lucene.Tests/Usings.cs:1]: This is already added in the `.csproj` and this file should be removed.

File: tests/Kentico.Xperience.Lucene.Tests/appsettings.json
Comment (seangwright) [inline tests/Kentico.Xperience.Lucene.Tests/appsettings.json:1]: What is this `appsettings.json` being used for?

If we need it, it shouldn't have a hardcoded connection string included in it.

File: Directory.Packages.props [hunk 1]
```diff
@@ -7,18 +7,22 @@
...
+    <PackageVersion Include="Kentico.Xperience.Cloud" Version="28.0.0" />
+    <PackageVersion Include="Kentico.Xperience.Imageprocessing" Version="28.0.0" />
     <PackageVersion Include="Kentico.Xperience.WebApp" Version="28.0.0" />
+    <PackageVersion Include="Kentico.Xperience.Core.Tests" Version="28.0.0" />
     <PackageVersion Include="Lucene.Net" Version="4.8.0-beta00016" />
     <PackageVersion Include="Lucene.Net.Analysis.Common" Version="4.8.0-beta00016" />
     <PackageVersion Include="Lucene.Net.Facet" Version="4.8.0-beta00016" />
-    <PackageVersion Include="SonarAnalyzer.CSharp" Version="9.6.0.74858" />
+    <PackageVersion Include="SonarAnalyzer.CSharp" Version="9.18.0.83559 " />
+    <PackageVersion Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.22.0" />
```
Comment (seangwright) [inline Directory.Packages.props:19]: Is this required to build the solution and run `DancingGoat` or tests?

File: tests/Kentico.Xperience.Lucene.Tests/Usings.cs
Comment (bkapustik) [reply tests/Kentico.Xperience.Lucene.Tests/Usings.cs:1]: Removed

File: tests/Kentico.Xperience.Lucene.Tests/appsettings.json
Comment (bkapustik) [reply tests/Kentico.Xperience.Lucene.Tests/appsettings.json:1]: Removed

File: Directory.Packages.props (hunk 1 above)
Comment (bkapustik) [reply Directory.Packages.props:19]: Reversed

**Review by seangwright — APPROVED** [review]

**Pull request merged**